mappings/net/minecraft/client/render/BuiltChunkStorage.mapping

```diff
@@ -1,7 +1,15 @@
 CLASS net/minecraft/unmapped/C_rqhfsngs net/minecraft/client/render/BuiltChunkStorage
+	FIELD f_afwynqyz pastViewDistance I
```

Comment:
`past` or `last`? i should look closer in the code later

Reply:
to note: mojang and yarn both name this `viewDistance`, no prefix

Reply:
yeah ngl i kinda guessed on this one